updatecli/updatecli.d/updatecli.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: "Bump Go version in updatecli"
+scms:
+  kine:
+    kind: "github"
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      username: "{{ requiredEnv .github.username }}"
+      token: '{{ requiredEnv .github.token }}'
+      owner: "{{ .kine.org }}"
+      repository: "{{ .kine.repo }}"
+      branch: "{{ .kine.branch }}"
```

Comment:
This "branch" is a bit unintuitive, this is the branch that will be checked out from source and targeted with PRs, but a new randomly generated branch will be created to host the change. see https://github.com/matttrach/kine/pull/1 as an example